extensions/amp-recaptcha-input/amp-recaptcha-input.md

```diff
@@ -28,6 +28,15 @@ limitations under the License.
 
 This extension adds a parameter containing a recaptcha response token when a parent `<form>` element submits. `amp-recaptcha-input` does this by creating an iframe to load the reCAPTCHA v3 api script using the provided site key, and calling `grecaptcha.execute` with the provided site key and action.
 
+### Prerequisite
+Like how you integrate reCaptcha on a regular web page, you will need go through steps as described in the [official documentation of reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3).
+High level, you will need to register a sitekey, and setup a server endpoint that can process the recaptcha signal sent from your AMP page.
```

Comment:
Is there a difference between "recaptcha" and "reCaptcha"? Otherwise I'd recommend consistent casing for this.

Reply:
renamed all occurrences to reCAPTCHA